content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,64 @@
 
 ### Coaches
 
 ## Quiz Officials
+- No comment other than “correct” or “incorrect” need be announced by the quiz officials in making a decision.  
+    - However, at the discretion of the answer judge and quizmaster or at the request of the captain, additional information may be given:
+        - If the additional information regarding the ruling is provided at the discretion of the AJ/QM, and not at the request of the captain, this is not considered an explanation of a ruling (which would invalidate a Challenge or Protest).
+        - If the captain requests additional information regarding the ruling, this is considered an explanation of a ruling, and invalidates a Challenge or Protest. Before replying to a request for explanation, the QM must ask all teams if they wish to Challenge.
 
 ### Quizmasters
+The Quizmaster should: 
+- Be thoroughly acquainted with all rules, procedures, and material of that quiz year.
+- Be impartial and render decisions consistently.
+- See that all officials are briefed regarding rules and procedures.
+- Read at a normal conversation rate.  Hesitating or reading too quickly must be avoided since either may cause inaccurate quizzing.
+    - Read all questions at the same rate of speed with the ability to stop reading the question at the exact moment a jump is made.
+- Keep the quiz moving at a consistent pace.  A quiz should not be rushed, but it is important to maintain the quickest pace which does not interfere with accuracy.
+- Be open to a challenge and be willing to evaluate whatever the captains or co- captains feel is in error.
+- Facilitate a positive, uplifting environment throughout the quiz.  Quizmasters should speak clearly and be enthusiastic about quizzing. To this end, quizzers should be called by name, the score should be announced often (when a scoreboard is not present), and bonuses and quiz-outs should be promptly recognized.
+
+The Quizmaster must:
+- Identify the question type of each question and begin every question with, “question number_____, question”.
+- Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary. 
+    - When an answer is in question (different than what is on the card), the quizmaster will discuss with the answer judge.
+- During the 30-second answering period use the following prompts:
+    - If a quizzer has provided all the information in the answer, except the identification of a pronoun present which is part of the answer, the quizmaster will ask the quizzer to identify the specific pronoun, (example: “Can you identify ‘He’?” or “Can you identify ‘They’?”) The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification.   A pronoun may need to be identified only if the identification (antecedent) is in context.
```

Comment:
Does this mean that if a quizzer jumps before the end of the question, provides all the information in the Answer, they should be prompted to identify the pronoun in the Answer? And upon doing so, the QM would say "More", since the quizzer is missing information in the Question?

Reply:
I think so. Maybe make this explicit with an example?

Reply:
hmmm, I'm not a fan. With the exception of "Quote is Complete" I want all uses of Again/More to come before any uses of "Identify he / what is your question"

Reply:
also, if "can you identify x" is in the current rule book as an "example", does that mean the QM could use other terminology?

Reply:
I want to burn "identify" with fire and have "clarify" rise from the ashes like a beautiful phoenix. However, to maintain FE, I will let this one go... for now...

Reply:
As do I. I mean, if "identify" is merely an example, we could add an additional example??

Reply:
Though I think that bit being prefaced by "example: " in the current rule book is a bug. It's absolutely intended to be the ONLY wording a QM can use.

Reply:
Yeah, I think this is a rulebook bug. To maintain FE, we have to rewrite it in v1 to be "identify" and stipulate it's the only thing QMs can say. Then we can burn it all down in v-next.